src/festim/exports/maximum_volume.py

```diff
@@ -27,5 +27,7 @@ def compute(self):
         """
         solution = self.field.solution
         indices = self.volume.locate_subdomain_entities(solution.function_space.mesh)
-        self.value = np.max(self.field.solution.x.array[indices])
+        max_value = max(self.field.solution.x.array[indices])
+        MPI.COMM_WORLD.barrier()
```

Comment:
1. I believe we still want to use numpy.max instead of native max here for performance purposes, no?
2. what does `barrier()` do?

Reply:
1. Yes sorry, I had just copied an example I had found. Now just using numpy within the `allreduce' function instead

2. It seems the barrier function forces all processes to reach the barrier before any of them can continue execution, however for this case it may not be necessary as all the processors will be following the same logical flow. Do you have any thoughts on this @jorgensd ?

Reply:
Plus shouldn't it be `solution.function_space.mesh.comm.barrier()` instead of comm world?

Reply:
I guess so